test/sqllogictest/table_func.slt

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# This file is derived from the logic test suite in CockroachDB. The
+# original file was retrieved on June 10, 2019 from:
+#
+# The original source code is subject to the terms of the Apache
+# 2.0 license, a copy of which can be found in the LICENSE file at the
+# root of this repository.
+
+statement ok
+CREATE TABLE y (a JSONB)
+
+# Ensure this does not panic.
+query TTTT
+SELECT * FROM y a, y b, jsonb_each(a.a);
+----
```

Comment:
Don't have the context on the rest of the change, but there's a bunch of other tests that use `jsonb_each` in jsonb.slt. (But if you have grand designs of testing more general SRF-specific things here rather than JSON-specific things, that makes sense too!)

Reply:
Yeah I thought about that, but it's not really what this bug was about, and I did intend to have more SRF changes incoming!